app/controllers/ordrs_controller.rb

```diff
@@ -0,0 +1,75 @@
+class OrdrsController < ApplicationController
+  before_action :set_ordr, only: [:show, :edit, :update, :destroy]
+
+  # GET /ordrs
+  # GET /ordrs.json
+  def index
+    @ordrs = Ordr.all
```

Comment:
Ordr is your Model name?????????????

Reply:
Yes , as i mistakenly added orders references to itself and i created a new scaffold with name ordr with the same functionality